### PR TITLE
Pbw 3181

### PR DESCRIPTION
**Pull request by chibzu**

### CODE REVIEW

**Comment by chibzu** [comment]
Should be merged with
https://github.com/ooyala/skin-config/pull/100

**Comment by mistermendez** [comment]
highlight volume on mouseover in controlBar-test.js fails ln:583

this is ready to merge after that is fixed

File: js/components/controlBar.js [hunk 1]
```diff
@@ -260,99 +218,95 @@ var ControlBar = React.createClass({
...
-      volumeIconSetting.msFilter = "";
     }
 
     var controlItemTemplates = {
-      "playPause": <div className="playPause" style={InlineStyle.controlBarStyle.controlBarItemSetting}>
-        <span className={playClass} style={iconSetting}
-          onClick={this.handlePlayClick} onTouchEnd={this.handlePlayClick} onMouseOver={this.highlight} onMouseOut={this.removeHighlight}>
+      "playPause": <button className="playPause controlBarItem" style={dynamicStyles.generalIconSetting} key="playPause">
+        <span className={playClass} style={dynamicStyles.iconCharacter}
+          onClick={this.handlePlayClick} onMouseOver={this.highlight} onMouseOut={this.removeHighlight}>
```
Comment (mistermendez) [inline js/components/controlBar.js]: we also need to move `onClick` from `span` to `button` element

File: js/components/controlBar.js [hunk 2]
```diff
@@ -260,99 +218,95 @@ var ControlBar = React.createClass({
...
       </div>,
 
-      "volume": <div className="volume" style={InlineStyle.controlBarStyle.controlBarItemSetting}>
-        <span className={muteClass} style={volumeIconSetting}
-          onClick={this.handleVolumeIconClick} onTouchEnd={this.handleVolumeIconClick}
-          onMouseOver={this.volumeHighlight} onMouseOut={this.volumeRemoveHighlight}>
+      "volume": <button className="volume controlBarItem" style={dynamicStyles.generalIconSetting}
+        onMouseOver={this.volumeHighlight} onMouseOut={this.volumeRemoveHighlight} key="volume">
+        <span className={muteClass} style={volumeIconSetting} ref="volumeIcon"
+          onClick={this.handleVolumeIconClick}>
```
Comment (mistermendez) [inline js/components/controlBar.js]: we also need to move `onClick` from `span` to `button` element

File: js/components/controlBar.js [hunk 3]
```diff
@@ -260,99 +218,95 @@ var ControlBar = React.createClass({
...
-      </div>,
+      </button>,
 
-      "closedCaption": <div className="closedCaption" style={InlineStyle.controlBarStyle.controlBarItemSetting}>
-        <span className={this.props.skinConfig.icons.cc.fontStyleClass} onMouseOver={this.highlight} onMouseOut={this.removeHighlight}
-          onClick={this.handleClosedCaptionClick} onTouchEnd={this.handleClosedCaptionClick} style={iconSetting}>
+      "closedCaption": <button className="closedCaption controlBarItem" style={dynamicStyles.generalIconSetting} key="closedCaption">
+        <span className={this.props.skinConfig.icons.cc.fontStyleClass} style={dynamicStyles.iconCharacter}
+          onMouseOver={this.highlight} onMouseOut={this.removeHighlight}
+          onClick={this.handleClosedCaptionClick}>
```
Comment (mistermendez) [inline js/components/controlBar.js]: we also need to move `onClick` from `span` to `button` element

File: js/components/controlBar.js [hunk 4]
```diff
@@ -260,99 +218,95 @@ var ControlBar = React.createClass({
...
-      </div>,
+      </button>,
 
-      "share": <div className="share" style={InlineStyle.controlBarStyle.controlBarItemSetting}>
-        <span className={this.props.skinConfig.icons.share.fontStyleClass} onMouseOver={this.highlight} onMouseOut={this.removeHighlight}
-          onClick={this.handleShareClick} onTouchEnd={this.handleShareClick} style={iconSetting}>
+      "share": <button className="share controlBarItem" style={dynamicStyles.generalIconSetting} key="share">
+        <span className={this.props.skinConfig.icons.share.fontStyleClass} style={dynamicStyles.iconCharacter}
+          onMouseOver={this.highlight} onMouseOut={this.removeHighlight}
+          onClick={this.handleShareClick}>
```
Comment (mistermendez) [inline js/components/controlBar.js]: we also need to move `onClick` from `span` to `button` element

**Comment by mistermendez** [comment]
looking good :cake:

**Pull request merged**